src/main/groovy/io/seqera/wave/service/builder/DockerBuildStrategy.groovy

```diff
@@ -113,7 +113,7 @@ class DockerBuildStrategy extends BuildStrategy {
                 ? cmdForKaniko( req.workDir, credsFile, spack, req.platform)
                 : cmdForSingularity( req.workDir, credsFile, spack, req.platform)
 
-        return dockerCmd + launchCmd(req)
+        return dockerCmd + builderName(req.buildId) +launchCmd(req)
```

Comment:
Is `builderName` really needed? could not that logic managed by `cmdForKaniko` and `cmdForSingularity` ?

Reply:
yes, this is not correct here, this builder name is only need when using docker, I am working on it